android/build.gradle

```diff
@@ -45,5 +45,7 @@ dependencies {
     implementation('com.google.android.exoplayer:extension-okhttp:2.18.1') {
         exclude group: 'com.squareup.okhttp3', module: 'okhttp'
     }
+    implementation 'com.google.android.exoplayer:extension-ima:2.11.4'
```

Comment:
Should be 2.18.1. can there be some side effects to add this dependency by default?

Reply:
Yes you're absolutely right, I forgot to push this file, sorry, I'll push it now